torch/lib/TensorLib/dispatch_macros.py

```diff
@@ -0,0 +1,40 @@
+from code_template import CodeTemplate
```

Comment:
This allows use to create templates for functions. One such use case is letting you have access to the scalar_type in your function.